node/pkg/processor/observation.go

```diff
@@ -192,7 +192,7 @@ func (p *Processor) handleObservation(ctx context.Context, m *gossipv1.SignedObs
 		// We have made this observation on chain!
 
 		// 2/3+ majority required for VAA to be valid - wait until we have quorum to submit VAA.
-		quorum := CalculateQuorum(len(gs.Keys))
+		quorum := vaa.CalculateQuorum(len(gs.Keys))
```

Comment:
Maybe you can use the new `verify` function so the logic is not repeated here and we only update it there?

Reply:
I don't think we have a VAA here.